### ISSUE TITLE
GitHub integration issues

### ISSUE THREAD

**Issue by ahmedre** [nijel/weblate#943]
hi,
thanks for writing weblate, it's a great tool!

i apologize in advance if this isn't a bug, but the documentation is a bit sparse about the github integration - so i've setup `hub` properly and can confirm that a `hub clone [person]/whatever` works fine with no prompts, etc.

when trying to add a component, i correctly see `GitHub` as one of the vcs options. I then enter a url such as `https://user:authkey@github.com/user/repo.git` (note that i also validate that a `hub clone` with that same url also works at the command line).

instead of working, i get back an error:
```
Failed to update repository: usage: git remote set-url [--push] <name> <newurl> [<oldurl>] or: git remote set-url --add <name> <newurl> or: git remote set-url --delete <name> <url> --push manipulate push URLs --add add URL --delete delete URLs (129)
```

if i change nothing except the vcs repository type to git, things work as expected. besides the required fields, the only thing i added was the file mask (for Android) and the base file. these work as expected when using a vcs of plain git.

i did set `GITHUB_USERNAME` in `weblate/settings.py` to the same username present in `~/.config/hub` in that generated file.

am i missing something?
thanks!

**Comment by nijel** [comment]
I've seen several complains on GitHub pull request integration, but I have no experience with it myself. It has been merged in #723.

**Comment by nijel** [comment]
The hub integration should be better since a6dbcc3ac4696703154a19e4ca42f81c5f15182f. It would be great if you could retest.
